### PR TITLE
+= docs path for developer documentaton and basic structure

### PR DESCRIPTION
**Pull request by Ishiido**
Anbei die Verzeichnisse für die Entwicklerdoku innerhalb des Projektes. Beste Grüße aus Berlin nach DD.

### CODE REVIEW

**Comment by Ishiido** [comment]
Ich hab doc/developer/* gelöscht und eine README dazugestellt, die sagt, dass in diesem Verzeichnis keine Docs mehr zu pflegen sind. Braucht man jetzt einen neuen Pull-Request?

**Comment by henning-gerhardt** [comment]
No, you did your changes on your branch which you use to open this pull request. So you did not open an other pull request.

Problem is now that your branch is really old against the current master so that changes on file `activemq_jms_api.md` in master are not in your branch. Can you please update your branch against current master so that merge conflicts could be solved?

**Comment by Ishiido** [comment]
sure. just wait.

**Comment by Ishiido** [comment]
so besser?

**Comment by henning-gerhardt** [comment]
I personally prefer `git rebase` to do such kind of work, but your way works too.

File: doc/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Information: This directory is not a documentation source any more. Developer docs are contained in
```
Comment (stweil) [inline doc/README.md]: What about removing the whole directory `doc` (including that new `doc/README.md`)?

Comment (henning-gerhardt) [inline doc/README.md]: I had this change already on my TODO list. I opened a pull request for removing this obsolete directory.

**Comment by Ishiido** [comment]
No problem with that. It contains an old structure for the user docs. If you consider this outdated just drop the "doc" dir.

File: doc/README.md (hunk 1 above)
Comment (Ishiido) [inline doc/README.md]: No problem with that. It contains an old structure for the user docs. If you consider this outdated just drop the "doc" dir.

**Pull request merged**